index.js

```diff
@@ -3,18 +3,23 @@
 	Author Tobias Koppers @sokra
 */
 var path = require("path");
-
 var loaderUtils = require("loader-utils");
 var validateOptions = require('schema-utils');
+var schema = require('./options.json')
+
+var validOptions = new WeakSet();
```

Comment:
Why?

Reply:
invoking `require()` is not needed within `pitch`, it should happen once.